doc/release_note/ommx-1.5.0.ipynb

```diff
@@ -0,0 +1,253 @@
+       "      <td></td>\n",
+       "      <td>[]</td>\n",
+       "      <td></td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>binary</td>\n",
+       "      <td>0.0</td>\n",
+       "      <td>0.0</td>\n",
+       "      <td>1.0</td>\n",
+       "      <td></td>\n",
+       "      <td>[]</td>\n",
+       "      <td></td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "      kind value lower upper name subscripts description\n",
+       "                                                        \n",
+       "id                                                      \n",
+       "1   binary   1.0   0.0   1.0              []            \n",
+       "2   binary   0.0   0.0   1.0              []            "
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "solution.decision_variables"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "From Python SDK 1.5.0, `Function` and its base classes, `Linear`, `Quadratic`, and `Polynomial` also support `evaluate` method:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "value=2.0, used_ids={1, 2}\n"
+     ]
+    }
+   ],
+   "source": [
+    "f = 2*x + 3*y\n",
+    "value, used_ids = f.evaluate({1: 1, 2: 0})\n",
+    "print(f\"{value=}, {used_ids=}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This returns evaluated value of the function and used decision variable IDs. In addition, there is `partial_evaluate` method"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "f2=Linear(3*x2 + 2), used_ids={1}\n"
+     ]
+    }
+   ],
+   "source": [
+    "f2, used_ids = f.partial_evaluate({1: 1})\n",
+    "print(f\"{f2=}, {used_ids=}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This creates a new function by substituting `x = 1`. `partial_evaluate` is also added to `ommx.v1.Instance` class:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "Function(x2 + 1)"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "new_instance = instance.partial_evaluate({1: 1})\n",
+    "new_instance.objective"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This method will be useful for creating a problem with fixing specific decision variables."
+   ]
+  }
```

Comment:
I suggest adding examples of cases where errors might occur to make it more helpful:

`````
A runtime error occurs if a non-existent variable ID is specified.

```python
# Example using a non-existent variable ID
try:
    value, used_ids = f.evaluate({3: 1})
except RuntimeError as e:
    print(f"KeyError: {e}")
```
`````

![image](https://github.com/user-attachments/assets/fc3b3b23-da07-49f6-883b-b747c25860d4)

Reply:
Added https://github.com/Jij-Inc/ommx/pull/206/commits/5a0b246bc874a30218ebfdf96c0315b028e54fb6